assets/src/components/form/validations/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * External imports
+ */
+import { __, _n, sprintf } from '@eventespresso/i18n';
+import {
+	isFloat as isValidFloat,
+	isInteger as isValidInteger,
+	isNumeric as isValidNumeric,
+	floatLessThanOrEqualTo,
+	stringLengthLessThanOrEqualTo,
+	numberLessThanOrEqualTo,
+	floatGreaterThanOrEqualTo,
+	stringLengthGreaterThanOrEqualTo,
+	numberGreaterThanOrEqualTo,
+} from '../../../data/helpers/validations';
```

Comment:
In the model-entity work, I put all the simple boolean type functions in the `eejs/utils` folder and then exposed them on the `eejs` global (`@eventespresso/eejs`).  I think for consistency we should do that here too.  `@eventespresso/helpers` (or `eejs.helpers`) are intended for more complex type helper functions.

However, in thinking about it, I think it may be useful to have all the simple `boolean` return type "validation" functions exposed on their own object on `eejs`.  So I think I'll start a pull that moves all these into their own `validators` object (see #765)

Reply:
ya I thought the `utils` folder made more sense, but only put them in the helpers because that's where you said to put them !
I agree that a separate validators object makes the most sense.

Reply:
I did? lol.  It's very possible at the time I thought that made sense and that was before I created the is-* things I put in `utils` with the model-entity stuff (it was really recentish I did that).  Regardless, I'ma gonna do that pull :)

Reply:
he he ya here: https://github.com/eventespresso/event-espresso-core/pull/745#discussion_r226701442

it happens